lib/diagnostic.js

```diff
@@ -6,29 +6,51 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (paragraph) {
+  let words = paragraph.split(' ');
```

Comment:
This should take the regular expression ```/\s+/``` as its argument so that it splits using any whitespace (spaces, tabs, line breaks, etc.) rather than only spaces.